deployment/clouddeploy/gke-workers/base/combine-to-osv.yaml

```
@@ -22,10 +22,10 @@ spec:
               privileged: true
             resources:
               requests:
-                cpu: 2
+                cpu: 30
                 memory: "1G"
               limits:
-                cpu: 2
+                cpu: 30
                 memory: "2G"
```

```suggestion
                memory: "8G"
```

Let's just bump memory up just in case we get bottlenecked on this, we are using a high mem node pool so this should not be a problem.